### PR TITLE
[Install.md] Further instructions for Windows compilation

### PR DESCRIPTION
**Pull request by marcnunezc**
Hi! I have been testing the Windows build myself and I have added some more comments that might be helpful for new users. Here are the list of changes/additions:

- Moved the "example scripts" section up, so it appears before the Post Compilation and Adding Application sections (feels more natural to me this way). 
- Visual Studio download link now points to 2019 version (it was pointing to 2017 earlier).
- Adding instruction on what workload packages to install with Visual Studio (we only need the C++ one)
- Adding instructions in case the user does not have the 2019 VS version.
- Adding more instructions to set/edit environment variables.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -69,9 +69,11 @@ Additionaly, Visual Studio is required to compile in Windows.
 
         *Visual Studio* is the only compiler officially supported to build *Kratos* under *Windows*. The minimium required version is Visual Studio 2017, but we recommend to use Visual Studio 2019 or higher. 
 
-        * [Download Visual Studio](https://visualstudio.microsoft.com/en/thank-you-downloading-visual-studio/?sku=Community&rel=15)
+        * [Download Visual Studio](https://visualstudio.microsoft.com/en/thank-you-downloading-visual-studio/?sku=Community&rel=16)
 
-        Since *Visual Studio* is a multi-language IDE, some distributions come without C++ compiler. Please, make sure that you can create a C++ project before continuing, in case C++ packages were missing you will be prompt to download them.
+        Since *Visual Studio* is a multi-language IDE, some distributions come without C++ compiler. Please, make sure that you can create a C++ project before continuing, in case C++ packages were missing you will be prompt to download them. You can install the <span style="color:red">Desktop development with C++</span> workload with the Visual Studio Installer to acquire all necessary depencencies to compile C++ projects.
```
Comment (philbucher) [inline INSTALL.md]: `<span style="color:red">` seems not to work

File: INSTALL.md [hunk 2]
```diff
@@ -323,6 +307,69 @@ rm -rf "${KRATOS_BUILD}/${KRATOS_BUILD_TYPE}/CMakeFiles"
...
+set PYTHONPATH=%PYTHONPATH%;C:/Kratos/bin/Release
+set PATH=%PATH%;C:/Kratos/bin/Release/libs
+```
+In *Windows Powershell*:
+```cmd
+$Env:PYTHONPATH+=";C:/Kratos/bin/Release"
+$Env:PATH+=";C:/Kratos/bin/Release/libs"
+```
+
+Or set them permanently using the  <span style="color:red">Edit the system environtment variables </span> option in the Control panel.
```
Comment (philbucher) [inline INSTALL.md]: same
```suggestion
Or set them permanently using the  <span style="color:red">Edit the system environment variables </span> option in the Control panel.
```

**Review by philbucher — COMMENTED** [review]
looks good from my side 👍 Thx

Since I am (not yet) a win expert I let @roigcarlo approve

File: INSTALL.md (hunk 1 above)
Comment (marcnunezc) [reply INSTALL.md]: True! I have replaced it with bold text

**Comment by roigcarlo** [comment]
Ok to me, I will approve asap (its giving me error 500, seems a github problem)

**Review by roigcarlo — APPROVED** [review]

**Pull request merged**